src/components/completing-party/CompletingParty.vue

```diff
@@ -73,27 +82,57 @@
     <!-- Summary Block -->
     <template v-else>
       <!-- Table Headers -->
-      <v-row class="list-header pb-3" no-gutters>
-        <v-col cols="12" sm="3">Name</v-col>
-        <v-col cols="12" sm="3">Mailing Address</v-col>
+      <v-row
+        class="list-header pb-3"
+        no-gutters
+      >
+        <v-col
+          cols="12"
+          sm="3"
+        >
+          Name
+        </v-col>
+        <v-col
+          cols="12"
+          sm="3"
+        >
+          Mailing Address
+        </v-col>
```

Comment:
It might be good to come to an agreement on when to make these multiple lines. I find the readability a lot nicer on one line in cases like this where its still short because it's easier to track the html structure. Normally I wouldn't mention this but it looks like you've made this change everywhere. Not sure how everyone else feels

Reply:
regardless in PRs like this where you have a LOT changing I think you should keep things like this out of it so its easier to review

Reply:
ah I guess this all changing due to the auto eslint? I see that makes more sense for this scenario. If that's the case ignore my comments

Reply:
I, too, prefer some things all on 1 line.

But, yes, eslint did this. I don't love it, but on the other hand it's consistent!

Duly ignoring... :)